dora/core/server/master/src/main/java/alluxio/master/metrics/CloudCostUtils.java

```diff
@@ -1,89 +1,89 @@
-/*
- * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
- * (the "License"). You may not use this work except in compliance with the License, which is
- * available at www.apache.org/licenses/LICENSE-2.0
- *
- * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
- * either express or implied, as more fully set forth in the License.
- *
- * See the NOTICE file distributed with this work for information regarding copyright ownership.
- */
-
-package alluxio.master.metrics;
-
-import alluxio.master.file.DefaultFileSystemMaster.Metrics.UFSOps;
-
-import com.google.common.collect.ImmutableMap;
-
-import java.util.Map;
-
-/**
- * This class contains utilities to compute cloud cost savings.
- */
-public final class CloudCostUtils {
-  private CloudCostUtils() {}  // prevent instantiation
-
-  /**
-   * Cost map is a map between ufs operations and their associated cost in dollars per operation.
-   * If it is a tiered storage solution, the top tier pricing is used.
-   */
-  // Aws pricing https://aws.amazon.com/s3/pricing/ last updated 02/19/2021
-  public static final Map<UFSOps, Double> S3_COSTMAP =
```

Comment:
maybe we can keep this and it might soon be useful in terms of data insight related functionalities